### ISSUE TITLE
Relax nbsp check for %description

### ISSUE THREAD

**Issue by jengelh** [rpm-software-management/rpmlint#554]
```
[   15s] libtest.spec:47: W: non-break-space line 47, char 20
[   15s] The spec file contains a non-break space, which looks like a regular space in
[   15s] some editors but can lead to obscure errors. It should be replaced by a
[   15s] regular space.
```

nbsp occuring in `%description` should be fine and no warning should be emitted.

**Comment by scarabeusiv** [comment]
As per your other comment we might avoid scanning the %changelog entries too as it should not affect the resulting output.
